src/OctoshiftCLI.Tests/Octoshift/ReclaimServiceTests.cs

```diff
@@ -437,7 +437,7 @@ public async Task ReclaimMannequins_NoTarget_No_Reclaim_IssuesError()
 
             _mockGithubApi.Setup(x => x.GetOrganizationId(TARGET_ORG).Result).Returns(ORG_ID);
             _mockGithubApi.Setup(x => x.GetMannequins(ORG_ID).Result).Returns(mannequinsResponse);
-            _mockGithubApi.Setup(x => x.GetUserId(TARGET_USER_LOGIN)).Returns(Task.FromResult<string>(null));
+            _mockGithubApi.Setup(x => x.GetUserId(TARGET_USER_LOGIN)).Throws(new OctoshiftCliException($"Could not resolve to a User with the login of '{MANNEQUIN_LOGIN}'."));
```

Comment:
This unit test wasn't realistic before - it didn't reflect the actual behaviour of `GetUserId`.